packages/components/src/Button/button.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { Story } from "@storybook/react/types-6-0";
 import React from "react";
-import Clickable from "../Clickable";
+import ClickableStyle from "../ClickableStyle";
```

Comment:
Another name idea just occurred to me: Actionable

...because buttons and links are the main way that action happens on a web site? 🤷

Reply:
How about `ActionableStyle`?! 😅 I think I'll stick with `ClickableStyle` for now, but I'm happy to revisit this if people think it's weird or if we decide we like something else better.